pkg/cluster/util.go

```diff
@@ -483,3 +484,13 @@ func (c *Cluster) GetSpec() (*acidv1.Postgresql, error) {
 func (c *Cluster) patroniUsesKubernetes() bool {
 	return c.OpConfig.EtcdHost == ""
 }
+
+func (c *Cluster) logLogicalBackupJobChanges(old, new *batchv1beta1.CronJob, reason string) {
```

Comment:
This is confusing, since the function is used only once, and doesn't really have any significant functionality.